src/plasoscaffolder/frontend/main.py

```diff
@@ -1,4 +1,7 @@
 # -*- coding: utf-8 -*-
+# pylint: disable=missing-docstring
+# because click does funny things with it
```

Comment:
`# because click does funny things with it`

Maybe:
```
# Docstrings are used by click in generating output. Since we don't want this behavior the following
# functions do not contain docstrings.
```